src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardElements.kt

```diff
@@ -482,13 +482,8 @@ private fun getEmptyLineDisplayPair() = listOf("<empty>" to HorizontalAlignment.
 private fun getIslandDisplayPair() =
     listOf("§7㋖ §a" + HypixelData.skyBlockIsland.displayName to HorizontalAlignment.LEFT)
 
-// TODO merge with LorenzUtils.skyBlockArea
 private fun getLocationDisplayPair() = buildList {
-    val location =
-        getGroupFromPattern(ScoreboardData.sidebarLinesFormatted, ScoreboardPattern.locationPattern, "location").trim()
```

Comment:
wouldnt this mean you can also remove this pattern from the code?

Reply:
Nope that's still needed for hiding the line